nsis/src/nsis/tests/NsisExecutableTest.java

```diff
@@ -81,6 +89,7 @@ public void testNsisCreationZlibCompressed() throws IOException, InvalidFormatEx
 		}
```

Comment:
Should we add the expected results to these tests as you add them above? I think they will be identical.

Same for the Bzip test below.

Reply:
Good point. Also the test I added for the pages only checks the number of pages. It does not check the inner values of a page, should I add some tests for that? And if so, how do I choose which value to test? I don't think we need to test all 12 parameters

Reply:
I believe you are right, we want to write "minimal" tests and not test each and every attribute.

We want to test "behaviours", in general.